### ISSUE TITLE
Google UID and domain change

### ISSUE THREAD

**Issue by mpeltonen** [reaktor/serve-static-git#7]
This app currently only supports configuring a single domain as allowed domain: https://github.com/reaktor/serve-static-git/blob/master/src/main.js#L99

So a code change is needed if we need to support both `reaktor.fi` and `reaktor.com` simultaneously. If single domain support is enough, then just `GOOGLE_OAUTH_ALLOWED_DOMAIN` env var change in Heroku, when Google switch is done, is enough.

**Comment by sauri1** [comment]
reaktor.fi and reaktor.com domains are needed both in same time. Also google id needs to be fetched from ldap instead of assuming uid is always "ldap id"@"google domain".

**Comment by tkurki** [comment]
This is pages.reaktor.com (despite the name) and doesn't do anything with the the user's identity.

**Comment by mpeltonen** [comment]
Yeah, I was already guessing that we need/should update all of the apps to accept logins from both of the domains.

**Comment by rjoonas** [comment]
`allowedDomains: config.GOOGLE_OAUTH_ALLOWED_DOMAINS.split(',')` 😎
